README.md

```diff
@@ -1,50 +1,40 @@
 
-
 # O QUE É a comuREDE?
 
-UM NEGÓCIO DE IMPACTO SOCIAL OPEN SOURCE
+UM NEGÓCIO DE IMPACTO SOCIAL OPEN SOURCE  
    > Para distribuir internet gratuita e serviços de IoT nas periferias
 
 [VEJA AQUI](https://www.facebook.com/comuREDE/videos/897710280400227/)
 
-
-# WHAT IS comuREDE?
-  
-AN OPEN SOURCE SOCIAL IMPACT BUSINESS
-   > To deliver free internet and IoT services in shanty towns
-
-[WATCH HERE](https://youtu.be/DjV8mJWbhzo)
-
-
 # COMO FUNCIONA?
-   
+
 ### core
 
-Versão 0.18 e 0.20 - back / frontend
+Versão 0.18 e 0.20 - back / frontend  
```

Comment:
Yeah, these trailing spaces makes sense but I find them hard to maintain (it doesn't show up for some people when editing).

What about adding a blank line? Like this:

```md
Versão 0.18 e 0.20 - back / frontend

PHP 5 / HTML, CSS e JAVASCRIPT
```